ant-bootstrap/tests/cli_integration_tests.rs

```diff
@@ -176,3 +182,45 @@ async fn test_test_network_peers() -> Result<(), Box<dyn std::error::Error>> {
 
     Ok(())
 }
+
+#[test]
+fn test_cli_add_peer() -> Result<()> {
+    let temp_dir = TempDir::new()?;
+    let cache_path = temp_dir.path().join("cache.txt");
+
+    let peer_addr = format!(
+        "/ip4/{}/udp/8080/quic-v1/p2p/12D3KooWRBhwfeP2Y4TCx1SM6s9rUoHhR5STiGwxBhgFRcw3UERE",
+        LOCAL_IP
+    );
+
+    // ... rest of the test ...
+    Ok(())
+}
```

Comment:
Agreed that this would be a nice feature! Maybe add a TODO comment in the tests here so we don't forget them as is?